create.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+#set -o xtrace
+# 
+# Patch EKS nodes
+# 
+
+if [ -z $1 ]; then
+    echo "Error: cluster variable is not set, I don't know which cluster you want to create"
+    exit 1
+fi
+
+export WORKSPACE=$1
+
+# build network and EKS masters
+pushd infra
+rm -rf .terraform
+terraform init -backend-config ../workspaces/$WORKSPACE/backend.cfg 
+terraform plan -input=false -var-file="../workspaces/$WORKSPACE/terraform.tfvars" 
+terraform apply -auto-approve -input=false -var-file="../workspaces/$WORKSPACE/terraform.tfvars" 
+
+# Configure local kubernetes config
+terraform output kubeconfig > ~/.kube/config-eks
```

Comment:
As I understand it these two lines do almost exactly the same thing. It seems unnecessary and redundant to create the separate config-eks file which you can already get from terraform. I suggest keeping only the second one.
```suggestion

```